documentation/adr/0018-update-release-notes-every-pr.md

```diff
@@ -0,0 +1,45 @@
+[<-previous](./0017-separate-absolute-and-relative-indexing-APIs-in-pixel-array.md) |
+next->
+
+# 18 - Update Release Notes on Every Pull Request
+
+Date: 2020-Nov-19
+
+## Status
+
+Accepted
+
+## Context
+
+Following the release of v3.5.0 of Horace,
+it was found that breaking changes were not effectively communicated to users.
+A discussion on the best way to communicate these changes,
+as well as the best way for developers to track such changes, was had.
+It was decided release notes provided a solution to this problem.
+
+The workflow for producing release notes needs to be decided.
```

Comment:
"The workflow for compiling the final release notes from these notes will be created separately and recorded [...somewhere...]"

Reply:
It's decided in this document. The `Decision` section below is addressing that workflow.